src/Microsoft.Kusto.ServiceLayer/Microsoft.Kusto.ServiceLayer.csproj

```diff
@@ -8,14 +8,15 @@
     <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
     <EnableDefaultNoneItems>false</EnableDefaultNoneItems>
     <EmbeddedResourceUseDependentUponConvention>false</EmbeddedResourceUseDependentUponConvention>
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <DefineConstants>$(DefineConstants);NETCOREAPP1_0;TRACE</DefineConstants>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <PreserveCompilationContext>true</PreserveCompilationContext>
     <DebugType>portable</DebugType>
     <RuntimeIdentifiers>$(ToolsServiceTargetRuntimes)</RuntimeIdentifiers>
 	<!-- TODO FIX THESE WARNINGS ASAP -->
```

Comment:
>ASAP [](http://example.com/codeflow?start=30&length=4)

promises... promises... :)

Reply:
Right? This is why I suggest people just file issues instead of leaving comments...